bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -459,7 +459,12 @@ <h2 class="tween">
         <div class="intro tween">
           {% if l10n_has_tag('mwc_2015') %}
             <h4>{{ _('A superior mobile experience') }}</h4>
-            <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+            <div class="play-button"> 
+              <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+              <small class="download-other">
+                <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+              </small>
```

Comment:
Wondering about consistency. The other link is yellow. Changing it to white would make it look like plain text no? I was sticking to the current link theme in that section. Should be changing that?

<img width="450" alt="screenshot 2015-09-14 15 34 40" src="https://cloud.githubusercontent.com/assets/190703/9859850/83c4d374-5af6-11e5-90b2-b877a30455ad.png">